e2e/charts/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -38,28 +38,63 @@ spec:
 {{ toYaml .Values.image.pullSecrets | indent 8 }}
       {{- end }}
       containers:
-        - name: {{ template "newrelic.name" . }}
-          image: {{ template "newrelic.image" . }}
-          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+        - name: kubelet-scraper
+          image: {{ .Values.scraperImage.repository }}:{{ .Values.scraperImage.tag }}
+          imagePullPolicy: {{ .Values.scraperImage.pullPolicy }}
           securityContext:
             {{- if .Values.privileged }}
             privileged: true
             {{- else }}
             runAsUser: {{ .Values.runAsUser | default 1000 }}
+            runAsGroup: {{ .Values.runAsGroup | default 2000 }}
```

Comment:
I think we should avoid hardcoded `default`s in templates and just make it so in the `values.yml`. I think helm users are already used to not remove stuff from the values as that could break the chart, so I generally write charts with that in mind.